doc/nrf/app_dev/device_guides/nrf54l/fota_update.rst

```diff
@@ -0,0 +1,236 @@
+
+.. note::
+   When building the application with MCUboot in direct-xip mode with revert mechanism support, the signed image intended for flashing is automatically marked as confirmed.
+   Without this configuration, the application will fail to boot.
+   It must, however, be disabled when building update images.
+
+Both the ``SB_CONFIG_MCUBOOT_MODE_DIRECT_XIP`` and ``SB_CONFIG_MCUBOOT_MODE_DIRECT_XIP_WITH_REVERT`` Kconfig options automatically build application update images for both slots.
+To read about the files that are built when the option is enabled, refer to the :ref:`app_build_mcuboot_output` page.
+
+.. fota_upgrades_over_ble_mcuboot_direct_xip_nrfcdm_note_start
+
+.. note::
+   Support for FOTA updates with MCUboot in the direct-xip mode is available since the following versions of the `nRF Connect Device Manager`_ mobile app:
+
+   * Version ``1.8.0`` on Android.
+   * Version ``1.4.0`` on iOS.
+
+.. fota_upgrades_over_ble_mcuboot_direct_xip_nrfcdm_note_end
+
+.. fota_upgrades_over_ble_mcuboot_direct_xip_information_end
+
+.. _ug_nrf54l_developing_ble_fota_steps_testing:
+
+Testing steps
+=============
+
+.. fota_upgrades_outro_start
+
+To perform a FOTA update, complete the following steps:
+
+.. fota_upgrades_over_ble_nrfcdm_common_dfu_steps_start
+
+1. Generate the DFU package by building your application with the FOTA support over Bluetooth Low Energy.
+   You can find the generated :file:`dfu_application.zip` archive in the build directory.
+
+   .. note::
+      For each image included in the DFU-generated package, use a higher version number than your currently active firmware.
+      Otherwise, the DFU target may reject the FOTA process due to a downgrade prevention mechanism.
+
+#. Download the :file:`dfu_application.zip` archive to your device.
+   See :ref:`app_build_output_files` for more information about the contents of update archive.
+
+   .. note::
+      nRF Connect for Desktop does not currently support the FOTA process.
+
+#. Use the `nRF Connect Device Manager`_ mobile app to update your device with the new firmware.
+
+   a. Ensure that you can access the :file:`dfu_application.zip` archive from your phone or tablet.
+   #. In the mobile app, scan and select the device to update.
+   #. Switch to the :guilabel:`Image` tab.
+   #. Tap the :guilabel:`SELECT FILE` button and select the :file:`dfu_application.zip` archive.
+   #. Tap the :guilabel:`START` button.
+
+      .. note::
+         When performing a FOTA update with the iOS app for samples using random HCI identities, ensure that the :guilabel:`Erase application settings` option is deselected before starting the procedure.
+         Otherwise, the new image will boot with random IDs, causing communication issues between the app and the device.
+
+   #. Initiate the DFU process of transferring the image to the device:
+
+      * If you are using an Android device, select a mode in the dialog window, and tap the :guilabel:`START` button.
```

Comment:
It will be nice to not use the word "device" to refer to phone or table. Else, it is required to be consistent throughout and say "Android/iOS device" every time the work "device" in a sentence refers to the mobile phone/table instead of nRF chip.

```suggestion
      * If you are using an Android phone/tablet, select a mode in the dialog window, and tap the :guilabel:`START` button.
```

Reply:
@annwoj Can you comment on that?

Reply:
@omkar3141 This request is more general than this PR. Tech writers asked to raise jira ticket for it and not touch this in that review.